Source/WebKit/UIProcess/WebAuthentication/fido/CtapAuthenticator.cpp

```diff
@@ -127,6 +135,16 @@ void CtapAuthenticator::continueMakeCredentialAfterResponseReceived(Vector<uint8
             receiveRespond(ExceptionData { ExceptionCode::InvalidStateError, "At least one credential matches an entry of the excludeCredentials list in the authenticator."_s });
             return;
         }
+        if (error == CtapDeviceResponseCode::kCtap2ErrKeyStoreFull) {
+            auto& options = std::get<PublicKeyCredentialCreationOptions>(requestData().options);
+            if (options.authenticatorSelection->requireResidentKey || options.authenticatorSelection->residentKey == ResidentKeyRequirement::Required)
+                observer()->authenticatorStatusUpdated(WebAuthenticationStatus::KeyStoreFull);
+            else if (!m_isKeyStoreFull) {
+                m_isKeyStoreFull = true;
```

Comment:
How does this ever get unset? Do we create a new CTAP authenticator for each new request?